openedx/features/marketplace/decorators.py

```diff
@@ -1,11 +1,18 @@
-from django.http import HttpResponseForbidden
+from django.http import HttpResponseRedirect
+from django.urls import reverse
 
 
 def has_affiliated_user(function=None):
     def inner(request, *args, **kwargs):
         user = request.user
+
+        if user.is_authenticated:
+            redirect_url = reverse('additional_information')
+        else:
+            redirect_url = "/login?next=%s" % request.path
```

Comment:
can we use `"/login?next={}".format(request.path)` instead because thats the format thats been used in philu app.